scripts/lib/encode.js

```diff
@@ -2,8 +2,8 @@ window.Encode = {}
 
 window.Encode.URL = {
   encode: value => encodeURIComponent(value),
-
-  decode: value => decodeURIComponent(value)
+  decode: value => decodeURIComponent(value),
+  decodePlus: value => decodeURIComponent(value.replaceAll('+',' '))
```

Comment:
A style issue

Space is needed after comma.

Reply:
Fixed now.